README.md

```diff
@@ -104,13 +104,22 @@ var pickles = Gherkin.compile(gherkinDocument, 'uri_of_the_feature.feature', uui
 
 ```go
 // Go
-// Download the package via: `go get github.com/cucumber/cucumber/gherkin/go`
+// Download the package via: `go get github.com/cucumber/cucumber/gherkin/go/v27`
 import (
   "strings"
-  gherkin "github.com/cucumber/cucumber/gherkin/go"
+  gherkin "github.com/cucumber/cucumber/gherkin/go/v27"
```

Comment:
URLs, have an extra `cucumber` that needs to be removed

`github.com/cucumber/gherkin/go/v27` instead of:
`github.com/cucumber/cucumber/gherkin/go/v27`

Reply:
removed extra cucumber in url